VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx

```diff
@@ -83,69 +83,67 @@ const VeteranStatusScreen: FC<VeteranStatusScreenProps> = () => {
   }
 
   return (
-    <LargePanel title={t('veteranStatus.title')} rightButtonText={t('close')} dividerMarginBypass={true} testID="veteranStatusTestID">
-      <Box backgroundColor={theme.colors.background.veteranStatus as BackgroundVariant} flex={1}>
-        <Box mx={theme.dimensions.gutter} alignItems="center" mt={theme.dimensions.standardMarginBetween}>
-          <VAIcon testID="VeteranStatusCardVAIcon" name={'Logo'} />
-          {/* <Box my={theme.dimensions.standardMarginBetween}>
-          //TODO: Put back PhotoUpload later after concerns have been met
-            <PhotoUpload width={100} height={100} />
-          </Box> */}
-          <Box my={theme.dimensions.formMarginBetween}>
-            <TextView textTransform="capitalize" mb={theme.dimensions.textIconMargin} variant="BitterBoldHeading" color="primaryContrast" testID="veteranStatusFullNameTestID">
-              {personalInfo?.fullName}
-            </TextView>
-            {accessToMilitaryInfo && (
-              <Box display="flex" flexDirection="row">
-                {getBranchSeal()}
-                <TextView ml={10} variant="MobileBody" color="primaryContrast" testID="veteranStatusBranchTestID">
-                  {branch}
-                </TextView>
-              </Box>
-            )}
-          </Box>
-        </Box>
-        <Box mx={theme.dimensions.gutter}>
-          {ratingIsDefined && (
-            <Box {...boxProps}>
-              <TextView variant="MobileBodyBold" color="primaryContrast">
-                {t('disabilityRating.title')}
-              </TextView>
-              <TextView variant="MobileBody" color="primaryContrast" testID="veteranStatusDisabilityRatingTestID">
-                {combinedPercentText}
+    <LargePanel title={t('veteranStatus.title')} rightButtonText={t('close')} dividerMarginBypass={true} removeInsets={true} testID="veteranStatusTestID">
+      <Box mx={isPortrait ? theme.dimensions.gutter : theme.dimensions.headerHeight} alignItems="center" mt={theme.dimensions.standardMarginBetween}>
```

Comment:
Why are we changing the X margin from 20 to 64 when in landscape? Was that defined somewhere?